### PR TITLE
new commit for pull

### PR DESCRIPTION
**Pull request by ZolotarenkoM**
Please see the template

### CODE REVIEW

File: cloud_auto.yaml [hunk 1]
```diff
@@ -32,9 +37,14 @@ Mappings:
   AWSRegionArch2AMI:
     us-east-1:
       HVM64: ami-0ff8a91507f77f867
+  Region2ARNPrefix:
+    us-east-1:
+      ARNPrefix: 'arn:aws:'
+  Region2ELBAccountId:
+    us-east-1:
+      AccountId: '127311923021'
```
Comment (Emperorex) [inline cloud_auto.yaml]: It isn't necessary to hardcode your AccountId and arn prefix. lets use built in variables AWS::Region, AWS::AccountId. See more in https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html
`Usage: !Ref "AWS::Region"`

File: cloud_auto.yaml [hunk 2]
```diff
@@ -79,12 +90,36 @@ Resources:
...
+  LogsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref S3LoggingBucket
+      PolicyDocument:
+        Version: '2008-10-17'
+        Statement:
+        - Sid: ELBAccessLogs20130930
+          Effect: Allow
+          Resource: !Join ['', [!FindInMap [Region2ARNPrefix, !Ref 'AWS::Region',
```
Comment (Emperorex) [inline cloud_auto.yaml]: Lets simplify this block. Try to change !Join to !Sub

File: cloud_auto.yaml [hunk 3]
```diff
@@ -104,3 +139,6 @@ Outputs:
   WebsiteURL:
     Description: URL for website
     Value: !Join ['', ['http://', !GetAtt [MyLoadBalancer, DNSName]]]
+  ELBAccessLogsBucket:
+    Description: DNS name of the S3 bucket storing ELB Access Logs
+    Value: !GetAtt [S3LoggingBucket, DomainName]
```
Comment (Emperorex) [inline cloud_auto.yaml]: Lets use single style. You can use instead
`Value: !GetAtt S3LoggingBucket.DomainName`

Comment (ZolotarenkoM) [reply cloud_auto.yaml]: fixed

File: cloud_auto.yaml (hunk 2 above)
Comment (ZolotarenkoM) [reply cloud_auto.yaml]: fixed

**Review by Emperorex — APPROVED** [review]
Changes looks good to me

**Pull request merged**